composer.json

```diff
@@ -10,7 +10,7 @@
         "cweagans/composer-patches": "~1.0",
         "drupal/console": "~0.10",
         "drupal/core": "~8.0.3",
-        "drupal/og": "dev-port-og-complex-widget-test",
+        "drupal/og": "dev-userGroupsFix",
```

Comment:
My previous remarks here were not fixed. This includes https://github.com/amitaibu/og/pull/166 which is not complete, it still needs a test.